README.md

```diff
@@ -1,555 +1,102 @@
-# ScubaGear <!-- omit in toc -->
-<p>
-        <a href="https://github.com/cisagov/ScubaGear/releases" alt="ScubaGear version #">
-        <img src="https://img.shields.io/badge/ScubaGear-v1.2.0-%2328B953?labelColor=%23005288" /></a>
-</p>
-
-Developed by CISA, ScubaGear is an assessment tool that verifies that a Microsoft 365 (M365) tenant’s configuration conforms to the policies described in the Secure Cloud Business Applications ([SCuBA](https://cisa.gov/scuba)) Security Configuration Baseline [documents](https://cisagov.github.io/ScubaGear/baselines).
```

Comment:
Is there a reason we link to the GitHub Pages version of [/ScubaGear/baselines](https://cisagov.github.io/ScubaGear/baselines) instead of [/baselines/README.md](https://github.com/cisagov/ScubaGear/blob/main/baselines/README.md)? All of the Table of Contents items link to the GitHub repo versions, so pointing this out for consistency purposes.

Reply:
Changed.